crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/generated_acir.rs

```diff
@@ -29,6 +29,15 @@ pub(crate) struct GeneratedAcir {
 }
 
 impl GeneratedAcir {
+    /// Creates a fresh GeneratedAcir
+    pub(crate) fn new() -> Self {
```

Comment:
nit, though I'll probably push a commit -- we can use Default

Reply:
`new` functions removed